examples/platform/silabs/BaseApplication.cpp

```diff
@@ -67,6 +67,10 @@
 #include <app/clusters/network-commissioning/network-commissioning.h>
 #include <platform/silabs/NetworkCommissioningWiFiDriver.h>
 #include <platform/silabs/wifi/WifiInterface.h>
+
+#if CHIP_CONFIG_ENABLE_ICD_SERVER
```

Comment:
You could define CHIP_CONFIG_ENABLE_ICD_SERVER as 0:
```suggestion
#if defined(CHIP_CONFIG_ENABLE_ICD_SERVER) && CHIP_CONFIG_ENABLE_ICD_SERVER
```

Reply:
`CHIP_CONFIG_ENABLE_ICD_SERVER` is always defined so we don't need the extra define check.